sentences/en/fan_HassTurnOff.yaml

```diff
@@ -24,8 +24,8 @@ intents:
 
       - sentences:
           - "<turn> off[ all][ the] fan[s][<in_here>]"
-          - "<turn>[ all][ the] fan[s]( off;<in_here>)"
-          - "<turn>[ all][ the] fan[s] off"
+          - "<turn>[ all][ the] fan[s] [to]( off;<in_here>)"
+          - "<turn>[ all][ the] fan[s][ to] off"
```

Comment:
@coderabbitai why isn't the sentence 'Set all the fans to off' being matched to this line?